aws-lc-rs/src/digest/sha.rs

```diff
@@ -109,7 +138,7 @@ pub static SHA384: Algorithm = Algorithm {
     output_len: SHA384_OUTPUT_LEN,
     chaining_len: SHA512_OUTPUT_LEN,
     block_len: SHA512_BLOCK_LEN,
```

Comment:
Nit: SHA384 has the same block/chaining length as SHA512, so the values configured are correct.  A comment here indicating that would make it clear that this is not a typo.